molgenis/ds-tidyverse
# chore: cran changes

## To test
- Code/doc review of changes. Check that the CRAN points have all been addressed
- Check that CI is all green

### CRAN comments
- [ ] The Description field is intended to be a (one paragraph) description of what the package does and why it may be useful. Please add more details about the package functionality and implemented methods in your Description text and not just repeat your title again.
- [ ] Please always write package names, software names and API (application programming interface) names in single quotes in title and description. e.g: --> 'DataSHIELD', etc... Please note that package names are case sensitive.
- [ ] Please provide a link to the used webservices (DataSHIELD) to the description field of your DESCRIPTION file in the form <http:...> or <[https:...]https:...> with angle brackets for auto-linking and no space after 'http:' and 'https:'.
- [ ] Please add \value to .Rd files regarding exported methods and explain the functions results in the documentation. Please write about the structure of the output (class) and also what the output means. (If a function does not return a value, please document that too, e.g. \value{No return value, called for side effects} or similar) 
- [ ] Missing Rd-tags: 
- checkPermissivePrivacyControlLevel.Rd: \value
- distinctDS.Rd: \value
